dataeng/resources/snowflake-collect-queue-depth.sh

```diff
@@ -6,7 +6,7 @@ cd $WORKSPACE/analytics-tools/snowflake
 make requirements
 
 python collect_queue_depth.py \
-    --key_path $KEY_PATH \
-    --passphrase_path $PASSPHRASE_PATH \
-    --automation_user $SNOWFLAKE_USER \
-    --account $SNOWFLAKE_ACCOUNT
+    --key_path $WORKSPACE/analytics-secure/$KEY_PATH \
+    --passphrase_path $WORKSPACE/analytics-secure/$PASSPHRASE_PATH \
+    --automation_user $WORKSPACE/analytics-secure/$SNOWFLAKE_USER \
+    --account $WORKSPACE/analytics-secure/$SNOWFLAKE_ACCOUNT
```

Comment:
You'll want to keep these two lines as they were.
```suggestion
    --automation_user $SNOWFLAKE_USER \
    --account $SNOWFLAKE_ACCOUNT
```

Reply:
Yeah sorry, I didn't see some of your comments appear until after I pushed that last commit, so I'm going to move some of those things you mentioned above to the job DSL file in my next commit